core/analytics-enabled/src/main/kotlin/com/wire/android/feature/analytics/AnonymousAnalyticsManagerImpl.kt

```diff
@@ -182,7 +182,7 @@ object AnonymousAnalyticsManagerImpl : AnonymousAnalyticsManager {
         coroutineScope.launch {
             mutex.withLock {
                 if (!isAnonymousUsageDataEnabled) return@withLock
-                anonymousAnalyticsRecorder?.recordView(screen)
+                anonymousAnalyticsRecorder?.recordView(screen.convertToCamelCase())
```

Comment:
is it really needed to convert this to camel case for countly ?
i mean is the screen names defined there or we can define them as we need on android ?

Reply:
it's for consistency, so we have clean on the board: route is snake_case and class name is camelCase.